cmd/server/upsClient.go

```diff
@@ -15,8 +15,34 @@ type upsClient struct {
 
 const BaseUrl = "http://localhost:8080/rest/applications"
 
+// Delete the variant with the given google key
+func (client *upsClient) deleteAndroidVariant(key string) bool {
+	variant := client.hasAndroidVariant(key)
```

Comment:
ok... that could be done a bit different too - but let's keep names for now - nvm